tests/test_utils_aws.py

```diff
@@ -30,6 +39,18 @@ def mock_urlopen(text, code=200):
     return m
 
 
+def requires_botocore(f: Callable) -> Callable:
```

Comment:
Since all functions here need botocore to run probably easier to just use:

https://docs.pytest.org/en/stable/reference/reference.html#pytest-importorskip